src/packs/serverless.ts

```diff
@@ -0,0 +1,279 @@
+import { CfnResource } from 'aws-cdk-lib';
+import { IConstruct } from 'constructs';
+import { NagPack, NagPackProps } from '../nag-pack';
+import { NagMessageLevel } from '../nag-rules';
+import {
+  apigw,
+  appsync,
+  cloudwatch,
+  eventbridge,
+  iam,
+  lambda,
+  sns,
+  sqs,
+  stepfunctions,
+} from '../rules';
+
+/**
+ * Serverless Checks are a compilation of rules to validate infrastructure-as-code template against recommended practices.
+ *
+ */
+export class ServerlessChecks extends NagPack {
+  constructor(props?: NagPackProps) {
+    super(props);
+    this.packName = 'Serverless';
+  }
+  public visit(node: IConstruct): void {
+    if (node instanceof CfnResource) {
+      this.checkLambda(node);
+      this.checkCloudwatch(node);
+      this.checkIAM(node);
+      this.checkApiGw(node);
+      this.checkAppSync(node);
+      this.checkEventBridge(node);
+      this.checkSNS(node);
+      this.checkSQS(node);
+      this.checkStepFunctions(node);
+    }
+  }
+
+  /**
+   * Check Lambda Resources
+   * @param node the CfnResource to check
+   * @param ignores list of ignores for the resource
+   */
+  private checkLambda(node: CfnResource) {
+    this.applyRule({
+      info: 'The Lambda function should have tracing set to Tracing.ACTIVE.',
```

Comment:
```suggestion
      info: 'The Lambda function does not have tracing set to Tracing.ACTIVE.',
```
`does not` was appropriate wording

Reply:
fixed